### PR TITLE
Fix code samples in the README.md

### PR DESCRIPTION
**Pull request by Biruntha**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,10 +61,9 @@ public function main() {
     json jsonPayload = { "ID":"redis", "Name":"redis1", "Address":"localhost", "port":8000, "EnableTagOverride":false };
     var serviceRegister = consulClient->registerService(jsonPayload);
     if (serviceRegister is boolean) {
-        test:assertEquals(serviceRegister, true, msg = "Failed to call registerService()");
+        io:println("Status of the service: ", serviceRegister);
```
Comment (MaryamZi) [inline README.md]: Shall we rename `serviceRegister` to `serviceRegistrationResponse` ?

```suggestion
        io:println("Status of service registration: ", serviceRegister);
```

File: consul/Module.md [hunk 2]
```diff
@@ -58,90 +58,91 @@ consul:Client consulClient = new(consulConfig);
 
 Register services in Consul with the given `jsonPayload`.
 ```ballerina
+json jsonPayload = {"ID":"redis", "Name":"redis1", "Address":"localhost", "port":8000, "EnableTagOverride":false};
```
Comment (MaryamZi) [inline consul/Module.md]:
```suggestion
json jsonPayload = { "ID": "redis", "Name": "redis1", "Address": "localhost", "port": 8000, "EnableTagOverride": false };
```

File: README.md (hunk 1 above)
Comment (MaryamZi) [reply README.md]: Same for the Module.md too.

File: consul/Module.md [hunk 3]
```diff
@@ -58,90 +58,91 @@ consul:Client consulClient = new(consulConfig);
...
+   io:println("Error: ", <string>serviceRegister.detail().message);
 }
 ```
 
 Get the details of the service with the given `serviceName`.
 ```ballerina
 var serviceDetails = consulClient->getService(serviceName);
 if (serviceDetails is consul:CatalogService[]) {
-   io:println(serviceDetails);
+   io:println("Details of a particular service: ", serviceDetails);
```
Comment (MaryamZi) [inline consul/Module.md]:
```suggestion
   io:println("Details of the service: ", serviceDetails);
```

Why does `getService` return an array though? Isn't it details of a single service?

File: consul/Module.md [hunk 4]
```diff
@@ -58,90 +58,91 @@ consul:Client consulClient = new(consulConfig);
...
+   io:println("Error: ", <string>serviceDetails.detail().message);
 }
 ```
 
 Register a check in Consul with the given `jsonCheck`.
 ```ballerina
 var checkRegister = consulClient->registerCheck(jsonCheck);
 if (checkRegister is boolean) {
-   io:println(checkRegister);
+   io:println("Status of the check: ", checkRegister);
```
Comment (MaryamZi) [inline consul/Module.md]:
```suggestion
   io:println("Status of the check registration: ", checkRegister);
```

We may also have to rename the variable to something like `checkRegistrationResponse`

File: consul/Module.md [hunk 5]
```diff
@@ -58,90 +58,91 @@ consul:Client consulClient = new(consulConfig);
...
+   io:println("Error: ", <string>checkRegister.detail().message);
 }
 ```
 
 Get the details of checks with the given `state`.
 ```ballerina
 var checkDetails = consulClient->getCheckByState(state);
 if (checkDetails is consul:HealthCheck[]) {
-   io:println(checkDetails);
+   io:println("Details of the  passing/critical state checks: ", checkDetails);
```
Comment (MaryamZi) [inline consul/Module.md]:
```suggestion
   io:println("Details of checks by state: ", checkDetails);
```

Comment (MaryamZi) [reply consul/Module.md]: Same comment as others for variable name, please check all names.

File: consul/Module.md [hunk 6]
```diff
@@ -58,90 +58,91 @@ consul:Client consulClient = new(consulConfig);
...
+   io:println("Error: ", <string>checkDetails.detail().message);
 }
 ```
 
 Create the entry in Consul with the given `keyName` and `value`.
 ```ballerina
 var keyRegister = consulClient->createKey(keyName, value);
 if (keyRegister is boolean) {
-   io:println(keyRegister);
+   io:println("Status of the key: ", keyRegister);
```
Comment (MaryamZi) [inline consul/Module.md]:
```suggestion
   io:println("Status of the key creation: ", keyRegister);
```

File: consul/Module.md [hunk 7]
```diff
@@ -58,90 +58,91 @@ consul:Client consulClient = new(consulConfig);
...
+   io:println("Error: ", <string>keyRegister.detail().message);
 }
 ```
 
 Read the key in Consul with the given `key`.
 ```ballerina
 var keyValue = consulClient->readKey(key);
 if (keyValue is consul:Value[]) {
-   io:println(keyValue);
+   io:println("Details of a particular key: ", keyValue);
```
Comment (MaryamZi) [inline consul/Module.md]:
```suggestion
   io:println("Details of the key: ", keyValue);
```

File: consul/Module.md [hunk 8]
```diff
@@ -58,90 +58,91 @@ consul:Client consulClient = new(consulConfig);
...
+   io:println("Error: ", <string>keyValue.detail().message);
 }
 ```
 
 Deregister services with the given `serviceId`.
 ```ballerina
 var serviceDeregister = consulClient->deregisterService(serviceId);
 if (serviceDeregister is boolean) {
-   io:println(serviceDeregister);
+   io:println("Status of the service: ", serviceDeregister);
```
Comment (MaryamZi) [inline consul/Module.md]:
```suggestion
   io:println("Status of service deregistration: ", serviceDeregister);
```

File: consul/Module.md [hunk 9]
```diff
@@ -58,90 +58,91 @@ consul:Client consulClient = new(consulConfig);
...
+   io:println("Error: ", <string>serviceDeregister.detail().message);
 }
 ```
 
 Deregister checks with the given `checkId`.
 ```ballerina
 var checkDeregister = consulClient->deregisterCheck(checkId);
 if (checkDeregister is boolean) {
-   io:println(checkDeregister);
+   io:println("Status of the check: ", checkDeregister);
```
Comment (MaryamZi) [inline consul/Module.md]:
```suggestion
   io:println("Status of the check deregistration: ", checkDeregister);
```

File: consul/Module.md [hunk 10]
```diff
@@ -58,90 +58,91 @@ consul:Client consulClient = new(consulConfig);
...
+   io:println("Error: ", <string>checkDeregister.detail().message);
 }
 ```
 
 Delete entries with the given `keyName`.
 ```ballerina
 var deleteKey = consulClient->deleteKey(keyName);
 if (deleteKey is boolean) {
-   io:println(deleteKey);
+   io:println("Status of the key: ", deleteKey);
```
Comment (MaryamZi) [inline consul/Module.md]:
```suggestion
   io:println("Status of the key deletion: ", deleteKey);
```

File: consul/Module.md (hunk 3 above)
Comment (Biruntha) [reply consul/Module.md]: As per the endpoint [1] and API doc [2], it is listing Nodes for Service as an array. 

[1] https://github.com/Biruntha/module-consul/blob/2e49ee51a6df6f188459849492b313307c9f4bbf/consul/consul_endpoint.bal#L80
[2] https://www.consul.io/api/catalog.html#list-nodes-for-service

**Review by MaryamZi — APPROVED** [review]

**Pull request merged**